gno.land/cmd/gnoland/start.go

```diff
@@ -234,9 +235,10 @@ func execStart(ctx context.Context, c *startCfg, io commands.IO) error {
 
 	// Create a top-level shared event switch
 	evsw := events.NewEventSwitch()
+	minGasPrices := cfg.Application.MinGasPrices
 
 	// Create application and node
-	cfg.LocalApp, err = gnoland.NewApp(nodeDir, c.skipFailingGenesisTxs, evsw, logger)
+	cfg.LocalApp, err = gnoland.NewApp(nodeDir, c.skipFailingGenesisTxs, evsw, logger, minGasPrices)
```

Comment:
I'm not a huge fan of adding more pomp to `NewApp`, but it's honestly out of the scope of this PR to clean it up

Please resolve this, just wanted to note we should tackle the node creation flow after the launch